docs/src/main/asciidoc/datasource-guide.adoc

```diff
@@ -223,7 +223,7 @@ AgroalDataSource dataSource1;
 AgroalDataSource dataSource2;
 --
 
-== Agroal Configuration Reference
+=== Agroal Configuration Reference
```

Comment:
Why? I think it was correct before.

Reply:
Good catch. This should not have slipped in.